gui/elements/_viewer.py

```diff
@@ -119,6 +148,13 @@ def add_layer(self, layer):
         if len(self.layers) == 1:
             self.reset_view()
 
+    def _new_markers(self):
+        if self.dimensions.max_dims == 0:
+            empty_markers = empty((0, 2))
```

Comment:
Looks like a symptom of a bigger problem here. We should address why we have to do this at all sometime.

Reply:
yup 100% plan to fix this, but was going to wait to a future PR. @jni and @royerloic had a good suggestion about being able to pass a markers layer a shape / size parameter to create an empty one of the right shape.